A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -89,6 +101,17 @@ class A32NX_FWC {
 
         const inhibOverride = this.memoFlightPhaseInhibOvrd_memo.write(recall, this.flightPhaseEndedPulse);
         SimVar.SetSimVarValue("L:A32NX_FWC_INHIBOVRD", "Bool", inhibOverride);
+
+        if (SimVar.GetSimVarValue("L:PUSH_AUTOPILOT_MASTERAWARN_L", "Bool") || SimVar.GetSimVarValue("L:PUSH_AUTOPILOT_MASTERAWARN_R", "Bool")) {
+            this.warningPressed = true;
+        } else {
+            this.warningPressed = false;
+        }
+        if (SimVar.GetSimVarValue("L:PUSH_AUTOPILOT_MASTERCAUT_L", "Bool") || SimVar.GetSimVarValue("L:PUSH_AUTOPILOT_MASTERCAUT_R", "Bool")) {
+            this.cautionPressed = true;
+        } else {
+            this.cautionPressed = false;
+        }
```

Comment:
**question**: What was the intent of this code? The variables are set but never used.

Reply:
because on future, I want to control caution on FWC